BUILD

```diff
@@ -77,35 +88,39 @@ nogo(
     config = ":nogo_config.json",
     vet = True,
     visibility = ["//visibility:public"],
-    deps = [
-        "@org_golang_x_tools//go/analysis/passes/asmdecl:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/assign:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/atomicalign:go_default_library",
-        # "@org_golang_x_tools//go/analysis/passes/cgocall:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/composite:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/copylock:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/deepequalerrors:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/errorsas:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/fieldalignment:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/framepointer:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/httpresponse:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/ifaceassert:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/loopclosure:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/lostcancel:go_default_library",
-        # "@org_golang_x_tools//go/analysis/passes/nilness:go_default_library", # template methods currently cause this analyzer to panic
-        # "@org_golang_x_tools//go/analysis/passes/shadow:go_default_library", # Everyone shadows `err`
-        "@org_golang_x_tools//go/analysis/passes/shift:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/sortslice:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/stdmethods:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/stringintconv:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/structtag:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/tests:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/testinggoroutine:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/unmarshal:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/unreachable:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/unsafeptr:go_default_library",
-        "@org_golang_x_tools//go/analysis/passes/unusedresult:go_default_library",
-        "@com_github_nishanths_exhaustive//:exhaustive",
+    deps = ["@com_github_nishanths_exhaustive//:exhaustive"] + [
```

Comment:
This doesn't feel necessary to me, could we just leave it as a list of analyzers without the programmatic creation?

Reply:
My original intention was to have both the analyzer name in the config file and the package name here to come from the same Starlark array. I.e.

```
MY_ANALYZERS = [
    "asmdecl",
    "assign",
    ...
]
```

However I realized that the "analyzer's name" and the "package's name" are only _almost_ the same. I.e. `copylock` vs `copylocks` 🙄. So I had to use 2 arrays instead and things turned ugly.

Your suggestion makes a lot more sense than my mumbo jumbo 👍.
Fixed.